### ISSUE TITLE
Is there a way to view tests in a folder structure?

### ISSUE THREAD

**Issue by v-kydela** [hbenl/vscode-mocha-test-adapter#75]
The folder structure of a project is flattened when viewing tests in the test explorer. This is very inconvenient for large projects. Is there a way for tests to be grouped together?

**Comment by kingsimba** [comment]
True, especially when multiple projects are added into a single VSCode workspace.
Tests should be grouped into project and then files, as did in Visual Studio 2019.

![image](https://user-images.githubusercontent.com/8407634/86600192-4fbc4f80-bfd2-11ea-8344-ef4310dedb86.png)

**Comment by hbenl** [comment]
Actually if you use a multi-root workspace in VS Code, the tests _will_ be grouped by root folder.
But I agree that more options for grouping tests are desirable and I am working on it. However, this requires some refactoring of the code, so it may take some time yet.

**Comment by shand-obs** [comment]
I could really use this feature right now.

**Comment by manish-makwana** [comment]
I would love this feature too. @hbenl happy to help with implementing this. Do you have any details on the refactoring required? (BTW love this project, just found and configured it - so much nicer than running `npm run test` in the console all the time!)
